app/controllers/repair_requests_controller.rb

```diff
@@ -34,7 +34,7 @@ def create
         format.html { redirect_to work_order_path(@repair_request.work_orders.first.reference), notice: ['Repair raised!'] }
       else
         @cautionary_contacts = Hackney::CautionaryContact.find_by_property_reference(@property.reference)
-        @keyfax_session = Hackney::KeyfaxSession.get_startup_url(current_page_url: new_property_repair_request_url(@property.reference))
+        @keyfax_session = Hackney::KeyfaxSession.create(current_page_url: new_property_repair_request_url(@property.reference))
```

Comment:
I'm very worried this didn't make the tests crash. Can you please investigate and fix them?

Reply:
we don't test the controllers, so should I add this to the feature spec?

Reply:
the feature spec should be crashing here

Reply:
I mean, the feature spec is a browser that do the same thing as a user. If it crashes for a user, then the test should as well.